getting-started-volunteer.md

```diff
@@ -0,0 +1,26 @@
+# I want to volunteer, where do I start?
+
+So you know what [we are about](https://github.com/djangonaut-space/pilot-program/blob/main/README.md), you have read our [code of conduct](https://github.com/djangonaut-space/pilot-program/blob/main/conduct.md), and you are interested in volunteering, so what now? First of all, thank you! Our community thrives because of people like you. 
+
+### Types of Volunteers
```

Comment:
Is there a reason to not use `##`? If it's all the same, we should use `##` so the html headers get generated in their proper order for accessibility purposes.